### PR TITLE
Triggers rn fix

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/48153

## Description
Replace trigger id with trigger name.

## Screenshots
This:
image

instead of:
image

### CODE REVIEW

**Review by dorschw — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
What about the changes we talked about today?

**Comment by ChanochShayner** [comment]
> What about the changes we talked about today?

This PR is blocked by - [This](https://github.com/demisto/demisto-sdk/pull/2065) PR.

**Comment by ChanochShayner** [comment]
@ShahafBenYakir @altmannyarden 
After the new design - 
image

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -133,8 +133,8 @@ def check_configuration_file(command, args):
 @pass_config
 def main(config, version, release_notes):
     config.configuration = Configuration()
-    import dotenv
-    dotenv.load_dotenv(Path(os.getcwd()) / '.env')  # Load a .env file from the cwd.
+    # import dotenv
+    # dotenv.load_dotenv(Path(os.getcwd()) / '.env')  # Load a .env file from the cwd.
```
Comment (ShahafBenYakir) [inline demisto_sdk/__main__.py:137]: Remove?

Comment (ChanochShayner) [reply demisto_sdk/__main__.py]: Sure, done

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**